### PR TITLE
[QUARKS-39] cleanup java warnings

### PR DESCRIPTION
**Pull request by dlaboss**
note: SplitWithEnumSample imports auto-reorganized by eclipse

### CODE REVIEW

File: api/topology/src/test/java/quarks/test/topology/TStreamTest.java [hunk 1]
```diff
@@ -292,6 +292,7 @@ public void testSplitWithNegativeOutputs() throws Exception {
     private enum LogSeverityEnum {
         ALERT(1), CRITICAL(2), ERROR(3), WARNING(4), NOTICE(5), INFO(6), DEBUG(7);
 
+        @SuppressWarnings("unused")
```
Comment (dlaboss) [inline api/topology/src/test/java/quarks/test/topology/TStreamTest.java]: @Cazen pls adjust this post merge if desired.

File: samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java [hunk 2]
```diff
@@ -63,6 +63,7 @@ public static void main(String[] args) throws Exception {
 
         // Split the stream by blood pressure category
         List<TStream<Map<String, Integer>>> categories = readings.split(6, new ToIntFunction<Map<String, Integer>>() {
+            private static final long serialVersionUID = 1L;
```
Comment (dlaboss) [inline samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java]: @queeniema pls adjust this post merge if desired.   You should be able to replace the `new` with a lambda fn and simplify things.  e.g., something like
```
  ... readings.split(6, tuple -> {
                        if (tuple.get("Systolic") < 120 && ...
                            // Normal
                            return 0;
                        }  else if ...
                        ...
                    });
```

**Comment by dlaboss** [comment]
The travis failure is unrelated to this change.  
WebSocketClientTest.testSslReconnect() failed.  Might be intermittent.
Merging and then will look more at that.

File: api/topology/src/test/java/quarks/test/topology/TStreamTest.java (hunk 1 above)
Comment (Cazen) [inline api/topology/src/test/java/quarks/test/topology/TStreamTest.java]: @dlaboss +1 LGTM Thank you :)

**Pull request merged**